fdbserver/CommitProxyServer.actor.cpp

```diff
@@ -1445,6 +1445,17 @@ void maybeAddTssMapping(GetKeyServerLocationsReply& reply,
 	}
 }
 
+void addTagMapping(GetKeyServerLocationsReply& reply, ProxyCommitData* commitData) {
+	for (const auto& [_, shard] : reply.results) {
+		for (auto& ssi : shard) {
+			auto iter = commitData->storageCache.find(ssi.id());
+			if (iter != commitData->storageCache.end()) {
```

Comment:
I suppose this should always be true. In that case, we could `ASSERT_WE_THINK(iter != commitData->storageCache.end())` instead.

Reply:
Done.